ApollosProject/apollos-apps
# Fix email auth crash

## ✏️ Description

_Things this PR does:_
Fixes an issue in `master` where logging in via email crashes the app.

_Things this PR doesn't do:__
Handles situations where the user has closed the app entirely, then opens the app with the link. 
We haven't handled this in master for a while now. I took a look at what it would take to fix this, and it was easy to cause more issues than I solved. 

Long term we need to find the correct config for the `confg.Screens.Auth`. It wasn't as simple as adding
```
      Auth: {
        screens: {
          IdentityVerificationConnected: 'auth',
        },
      },
```      


## 🔬 To Test
1. Open the Apollos Demo app
2. Sign Out
3. Visit the Login screen.
4. Enter an email address and either login or register. 
5. Copy the link in your email and paste into this command and run `xcrun simctl openurl booted 'LINK FROM EMAIL HERE'`
6. Click "Login" and ensure you are logged in.

## 📸 Screenshots

https://user-images.githubusercontent.com/1637694/205706358-27eb8f97-e499-4fe5-ab87-eed026727d5e.mp4

## ⚠️ To-do Before Merge

n/a

## 📝 Checklist
